src/guides/v2.3/get-started/create-integration.md

```diff
@@ -29,7 +29,7 @@ To create an integration, follow these general steps:
 
 To develop a module, you must:
 
-1. **Create the module file structure.** The module for an integration can be placed anywhere under the Magento root directory, but the recommended location is `<magento_base_dir>/vendor/<vendor_name>/module-<module_name>`.
+1. **Create the module file structure.** The module for an integration can be placed anywhere under the Magento root directory, but the recommended location is `<magento_base_dir>/app/code/<vendor_name>/module-<module_name>`.
```

Comment:
The `module-` prefix is redundant in case of a non-composer installation. I'm not sure it would be even properly fetched by the autoloader. Have you tried to create a module with such naming under `app/code`?

```suggestion
1. **Create the module file structure.** The module for an integration can be placed anywhere under the Magento root directory, but the recommended location is `<magento_base_dir>/app/code/<vendor_name>/<module_name>`.
```